pkg/minikube/constants/constants.go

```diff
@@ -114,6 +114,8 @@ const (
 
 	// TimeFormat is the format that should be used when outputting time
 	TimeFormat = time.RFC1123
+	// NoLimit is the value that can be passed into the memory and cpus flags to specify to use maximum resources
+	NoLimit = "nolimit"
```

Comment:
how about `no-limit` or maybe even `max`

Reply:
Changed to max, much more instinctive name for it